tox.ini

```diff
@@ -6,15 +6,16 @@ passenv = https_proxy http_proxy no_proxy
 deps=
     six
     pytest
+    pytest-cov
     html5lib
     mock
     enum34
     typing
 setenv =
     SPHINX_TEST_TEMPDIR = {envdir}/testbuild
-    PYTHONDONTWRITEBYTECODE = true
```

Comment:
I noticed this line now. Why did you remove it? I don't opposite for this, but I'd like to know reason.

Reply:
From [here](https://github.com/sphinx-doc/sphinx/pull/4117/commits/6086d7b608a365dc3ccfcb37d8bac7f02de338fe):

> As I've previously discovered [1], this doesn't actually do anything in
> modern tox. Simply remove it.
>
> [1] https://review.openstack.org/#/c/369986/

I put all my rationale in my commit messages. Maybe I should start copy-pasting them into the pull request message? :smile:

Reply:
I understood. I will read your commit message in next time :-)